controller_manager/test/test_resource_manager.cpp

```diff
@@ -221,3 +224,151 @@ TEST_F(TestResourceManager, initialization_with_wrong_urdf) {
     SUCCEED() << e.what();
   }
 }
+
+TEST_F(TestResourceManager, initialization_with_urdf_unclaimed) {
+  auto urdf = urdf_head_ + test_hardware_resource_system_ + urdf_tail_;
+  // we validate the results manually
+  controller_manager::ResourceManager rm(urdf);
+
+  auto command_interface_keys = rm.command_interface_keys();
+  for (const auto & key : command_interface_keys) {
+    EXPECT_FALSE(rm.command_interface_is_claimed(key));
+  }
+  // state interfaces don't have to be locked, hence any arbitrary key
+  // should return false.
+  auto state_interface_keys = rm.state_interface_keys();
+  for (const auto & key : state_interface_keys) {
+    EXPECT_FALSE(rm.command_interface_is_claimed(key));
+  }
+}
+
+TEST_F(TestResourceManager, resource_claiming) {
+  auto urdf = urdf_head_ + test_hardware_resource_system_ + urdf_tail_;
+  controller_manager::ResourceManager rm(urdf);
+
+  const auto key = "joint1/position";
```

Comment:
you seem to be testing the "joint1/position" command interface here, then all the others plus this in a loop. Leftover?

Reply:
the loop has these values listed multiple times, as I want to verify the repeatability of resource claiming.